specification/dartLangSpec.tex

```diff
@@ -5204,28 +5204,42 @@ \subsubsection{Correct Member Overrides}
   $F$ must then be a subtype of $F'$.
```

Comment:
Should it have type `Object?` instead of `Object`, or is the spec just not null safe yet?

Reply:
I haven't started putting null-safety elements into the spec yet, we only have those snippets of the grammar where the null safety elements have been there for many months.